### PR TITLE
Feature: systemd n processes

### PR DESCRIPTION
**Pull request by lsh-0**
the lax 16.04 branch uses this and it appears to work. there is an unrelated newrelic error in there preventing it from starting but you can load this up for yourself with:

`./bldr masterless.set_versions:lax--mltest2,builder-base-formula@feat-systemd-n-processes,lax-formula@feat-ubuntu-1604`

### CODE REVIEW

File: elife/templates/systemd-multiple-processes.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+NUMBER={{ number }}
+timeout=120
+    echo "--------"
```
Comment (giorgiosironi) [inline elife/templates/systemd-multiple-processes.sh]: not sure there is a reason anymore for the additional indentation?

**Comment by giorgiosironi** [comment]
Output, first time when not running, second time when already running:
```
elife@mltest2--lax:~$ sudo /opt/bot-lax-adaptors.sh                                                                                        
--------
Current status of 1 bot-lax-adaptor processes
Now is 2017-08-29T13:28:52+00:00
● bot-lax-adaptor@1.service - "bot-lax-adaptor (singular)"
   Loaded: loaded (/lib/systemd/system/bot-lax-adaptor@.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
Stopping 1 bot-lax-adaptor processes
Now is 2017-08-29T13:28:52+00:00
Waiting for bot-lax-adaptor 1 to stop
Now is 2017-08-29T13:28:52+00:00
   Active: inactive (dead)
Stopped all bot-lax-adaptor processes
Starting 1 bot-lax-adaptor processes
Started 1 bot-lax-adaptor processes
elife@mltest2--lax:~$ sudo /opt/bot-lax-adaptors.sh 
--------
Current status of 1 bot-lax-adaptor processes
Now is 2017-08-29T13:28:59+00:00
● bot-lax-adaptor@1.service - "bot-lax-adaptor (singular)"
   Loaded: loaded (/lib/systemd/system/bot-lax-adaptor@.service; disabled; vendor preset: enabled)
   Active: active (running) since Tue 2017-08-29 13:28:52 UTC; 6s ago
 Main PID: 20653 (python)
   CGroup: /system.slice/system-bot\x2dlax\x2dadaptor.slice/bot-lax-adaptor@1.service
           └─20653 python src/adaptor.py --type sqs
Stopping 1 bot-lax-adaptor processes
Now is 2017-08-29T13:28:59+00:00
Waiting for bot-lax-adaptor 1 to stop
Now is 2017-08-29T13:28:59+00:00
   Active: inactive (dead)
Stopped all bot-lax-adaptor processes
Starting 1 bot-lax-adaptor processes
Started 1 bot-lax-adaptor processes
```

File: elife/templates/systemd-multiple-processes.sh (hunk 1 above)
Comment (lsh-0) [reply elife/templates/systemd-multiple-processes.sh]: quite right, I think I removed some enclosing conditional

**Comment by lsh-0** [comment]
that would be the verbose `systemctl status <process>`. we can make that less verbose if need be. I'm not 100% certain about this code, but it's a straight port, the logic isn't complex and nothing is using 16.04 yet so we can debug it later if anything presents itself.

**Pull request merged**